wikipedia_name_query/query.py

```diff
@@ -0,0 +1,48 @@
+'''
+The module SPARQLWrapper is used to send and format the sql query 
+'''
+
+from SPARQLWrapper import SPARQLWrapper, JSON
+
+
+
+class Query():
+    '''
+    This is a class used to hold the query
+    '''
+    @staticmethod
+    def get_person_info(person_name):
+        '''
+        This function querys dhpedia for the emtered person and retreives the data needed,
+        this is then put into a format and ready to be output
+        '''
```

Comment:
Need to add the `person_name` to the docstring:

```suggestion
    def get_person_info(person_name):
        '''
        This function querys dhpedia for the emtered person and retreives the data needed,
        this is then put into a format and ready to be output
        
        Parameters
        -----------
        person_name : str
            The name of the person to query
        '''
```

Review other functions for similar changes